src/DropDown/DropDown.component.tsx

```diff
@@ -61,10 +62,10 @@ const StyledDropDownContainer = styled(DropDownContainer)<
     position: absolute;
     padding: 0.25rem 0;
     z-index: 1;
-    background-color: neutrals.white.base;
+    background-color: ${th.color('neutrals.white.base')};
```

Comment:
This one shouldn't be necessary

Reply:
I believe I did this because my setup of VS Code has issues with understanding what's going on with xstyled, and I wanted formatting to work so I could better read/understand the component.